playbooks/azure/openshift-cluster/build_image.yml

```diff
@@ -0,0 +1,126 @@
+---
+# Note: this required ansible-galaxy install Azure.azure_modules
+
+- import_playbook: prerequisites.yml
+
+# create all the items necessary to make the VM
+- name: "Create resource group"
+  hosts: localhost
+  gather_facts: yes
+  tasks:
+  - name: provision public ip address
+    import_role:
+      name: openshift_azure
+      tasks_from: provision_instance.yml
+    vars:
+      openshift_azure_vm_base_image:
+        name: '{{ openshift_azure_base_image }}'
+        resource_group: '{{ openshift_azure_base_image_ns }}'
+  - name: get the openshift registry token
```

Comment:
why is this needed?

Reply:
The registry requires a token so we are logging in and getting the registry token for 30 days.

Reply:
Ah, of course.  This wouldn't be needed on CentOS, and indeed, now that 3.9 has shipped, I'm not sure it's needed directly here either? (we need to clear up the references to reg-aws in acs-engine too)